doc/OnlineDocs/contributed_packages/index.rst

```diff
@@ -22,8 +22,9 @@ Contributed packages distributed with Pyomo:
    mcpp.rst
    mindtpy.rst
    satsolver.rst
+   community.rst
 
 Contributed packages distributed independently of Pyomo, but accessible
 through ``pyomo.contrib``:
 
-* `pyomo.contrib.simplemodel <http://pyomocontrib-simplemodel.readthedocs.io/en/latest/source.html>`_
+* `pyomo.contrib.simplemodel <http://pyomocontrib-simplemodel.readthedocs.io/en/latest/source.html>`_
```

Comment:
I don't see the difference. Did you add a windows EOL here?

Reply:
Hmm, I don't think so (or at the very least it shouldn't be there) - I was playing around with this file when git kept telling me I had an unresolved conflict (which I later realized was normal)